upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -186,15 +186,20 @@ func (c *ApplyClusterCmd) Run() error {
 		}
 
 		if usesCNI(cluster) {
-			defaultCNIAsset := fmt.Sprintf("https://storage.googleapis.com/kubernetes-release/network-plugins/cni-8a936732094c0941e1543ef5d292a1f4fffa1ac5.tar.gz")
+			// TODO: we really need to sort this out:
```

Comment:
Do we need an issue to follow up on this in general? Wondering if we should track the future fix here independently